components/browser/state/src/main/java/mozilla/components/browser/state/state/BrowserState.kt

```diff
@@ -11,11 +11,14 @@ import mozilla.components.lib.state.State
  *
  * @property tabs the list of open tabs, defaults to an empty list.
  * @property selectedTabId the ID of the currently selected (active) tab.
+ * @property completedDownloads a set of ids of the completed downloads.
  * @property customTabs the list of custom tabs, defaults to an empty list.
  */
 data class BrowserState(
     val tabs: List<TabSessionState> = emptyList(),
     val selectedTabId: String? = null,
 
```

Comment:
nit: not sure why we had a empty line here.  Let's remove this line and also the extra empty line you added.